### DISCUSSION TITLE
Welcome to AI Habitat!

### DISCUSSION THREAD

**Discussion by aclegg3** [facebookresearch/habitat-lab#1024] [Announcements]
:wave: :smile: 
Welcome to the official community discussions forum for AI Habitat, supporting both [habitat-sim](https://github.com/facebookresearch/habitat-sim) and [habitat-lab](https://github.com/facebookresearch/habitat-lab) repositories. 

While here, we hope you will:
- Ask questions for the developers in "Q&A".
- Share your own work and research using Habitat in "Show and Tell".
- Brainstorm and explore the possibilities in "Ideas".
- Engage with the community in "General".

Please help us to foster a supportive and inclusive community environment by respecting our [code of conduct](https://opensource.fb.com/code-of-conduct/) and welcoming others with an open mind. We are the community we build together! :grinning: 

Have a bug to report or a feature enhancement to suggest? Consider opening an issue on [habitat-sim](https://github.com/facebookresearch/habitat-sim/issues) or [habitat-lab](https://github.com/facebookresearch/habitat-lab/issues) repositories. 

:tada:  Welcome to AI Habitat! :tada:

<img width="10%" height="10%" src="https://user-images.githubusercontent.com/1445143/205106177-e0d7040a-2578-4b52-97a5-fddd75159cf8.png">

For documentation, project context, and tutorials please visit us at https://aihabitat.org/.

**Comment by dhruvbatra** [comment]
Glad to see we're consolidating all public discussions here on github (as opposed to splitting some here and some on Slack).
